packages/snap/src/config.ts

```diff
@@ -50,41 +50,71 @@ enum CHAINS {
   Goerly = 5,
 }
 
-export const SUPPORTED_CHAINS: Record<number, ChainInfo> = {
+export const SUPPORTED_CHAINS: Record<number, ChainZkpInfo> = {
   [CHAINS.Polygon]: {
-    id: CHAINS.Polygon,
-    rpcUrl: 'https://polygon-rpc.com',
-    stateContractAddress: '0xf9bA419ad9c82451d31d89917db61253a1e46B3C',
+    targetChainId: CHAINS.Polygon,
+    targetRpcUrl: 'https://polygon-rpc.com',
+    targetStateContractAddress: '0xf9bA419ad9c82451d31d89917db61253a1e46B3C',
+
+    rarimoApiUrl: config.RARIMO_CORE_URL.mainnet,
```

Comment:
it could be shortened by spreading `mainnet` and `testnet` objects with these values, but teoretically there could be a bunch of different `rarimo` chains which is supported by different `target` chains, so mb let's just keep it simple?

Reply:
Duplicating values like this looks very error-prone and unscalable. Let's do something more elagant